lib/matplotlib/artist.py

```diff
@@ -1394,6 +1488,10 @@ def aliased_name_rest(self, s, target):
         alias, return 'markerfacecolor or mfc' and for the transform
         property, which does not, return 'transform'.
         """
+        # workaround to prevent "reference target not found"
+        if target in self._NOT_LINKABLE:
```

Comment:
Are non-linkable targets basically the methods defined on a private parent? If so, perhaps you can just check whether the class name starts with an underscore, rather than hard coding the entire list.

Reply:
That's what I first thought (and checked with the print statement). Turns out it is more complicated than that.

As a general statement, a setter is not linkable if it has not doc entry. All non-private classes can (and now do) write documentation entries setters. We don't autodoc private classes, so there is no default place to document their setters. However, sublcasses may reimplement the setter and document that, or explicit entries can be created (e.g. for `_AxesBase` setters).

I'm not even sure this is the whole story. Anyway, I've chosen to only explicitly backout for the properties that sphinx was complaining about. The other ones are found one way or the other.